docs/source/posts/2022/2022-09-28-week-14-shivam.rst

```diff
@@ -0,0 +1,45 @@
+Week 14 - Morphing is here! 
+===========================
+
+.. post:: September 28 2022
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- This week, I started with Multiple actor support in skinning, Managed to do it
+  successfully (see brainStem Model)
+
+- Implementing multiple animation channels support (as seen in the ``Fox`` model). The ``get_skin_timelines()`` method now returns a dictionary of all animation channels with Timeline as their value.
```

Comment:
Same as above.
Add a link to a video or add it here so that reader would get an idea about what you are specifying.

Reply:
Multiple animation channels are in the Fox model itself, maybe adding the link to the Fox model can help